firefox_puppeteer/ui/tabbar.py

```diff
@@ -20,7 +20,7 @@ class Tabs(BaseLib):
     @property
     def newtab_button(self):
         """
-        :returns: The new tab button element.
+        :returns: The new tab button element
```

Comment:
All those changes in this module will be invalid once I landed my big tabbar refactoring patch.